autoconf in devel/hwloc2 expects self-contained header.

In file included from conftest.c:180:
In file included from /usr/local/include/level_zero/zes_api.h:18:
/usr/local/include/level_zero/ze_api.h:7026:5: error: unknown type name 'bool'
    bool sRGB;                                      ///< [in] Is sRGB.
    ^

--- include/ze_api.h.orig	2021-11-05 23:01:25 UTC
+++ include/ze_api.h
@@ -7023,7 +7023,7 @@ typedef struct _ze_srgb_ext_desc_t
     ze_structure_type_t stype;                      ///< [in] type of this structure
     const void* pNext;                              ///< [in][optional] must be null or a pointer to an extension-specific
                                                     ///< structure (i.e. contains sType and pNext).
-    bool sRGB;                                      ///< [in] Is sRGB.
+    ze_bool_t sRGB;                                 ///< [in] Is sRGB.
 
 } ze_srgb_ext_desc_t;
 
